### PR TITLE
Add addMoreActionsButtons Hook (standardisation)

### PR DESCRIPTION
**Pull request by defrance**

### CODE REVIEW

File: htdocs/projet/tasks/task.php [hunk 1]
```diff
@@ -484,55 +484,60 @@
 			dol_fiche_end();
 		}
 
-
-		if ($action != 'edit')
+		$parameters = array();
```
Comment (simnandez) [inline htdocs/projet/tasks/task.php]: This must be in div tabsAction

Comment (eldy) [inline htdocs/projet/tasks/task.php]: Confirmed. This execute hook should be after the print div tabsAction

Comment (defrance) [inline htdocs/projet/tasks/task.php]: No, the additionnal button must be with the other button like in the other card page

Comment (eldy) [inline htdocs/projet/tasks/task.php]: @Sure, that's why it must be into the div.
All buttons are into the <div tabsAction
All
print '<a class="butAction"
and
hooks to add button should be inside the
<div tabsAction
This div means "begin part with buttons".

**Pull request merged**